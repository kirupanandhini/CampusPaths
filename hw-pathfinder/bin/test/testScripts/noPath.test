# Find a path with Dijkstra's algorithm when
# the path found by BFS is different.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

FindPath simpleGraph A B
