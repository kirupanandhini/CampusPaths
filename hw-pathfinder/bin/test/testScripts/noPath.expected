# Find a path with Dijkstra's algorithm when
# the path found by BFS is different.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

path from A to B:
no path found
